### ISSUE TITLE
Multi-select Darkmode not working

### ISSUE THREAD

**Issue by robs007** [livewire/flux#834]
### Flux version

1.0.29

### Livewire version

3.5.17

### What is the problem?

Installed a fresh laravel version with jetstream dark mode. When I add a flux select with the multiple attribute, I get a white box for the select. The dropdown is actually in dark mode.
Removing the multiple attribute the select works fine.

![Image](https://github.com/user-attachments/assets/a703bdba-9daa-4452-b553-be1d1cef3e2b)

These CSS elements seem to be the issue.
![Image](https://github.com/user-attachments/assets/8ed5ce26-519a-41f6-9383-231ac6b0d5e8)

![Image](https://github.com/user-attachments/assets/38a39f3b-97dd-4b88-91f5-8fe8fa070123)

### Code snippets

```
 <div class="py-12">
        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
            <div class="bg-white dark:bg-gray-800 overflow-hidden shadow-xl sm:rounded-lg">
                {{--                <x-welcome />--}}
                <div class="w-44 p-5">
                    <flux:select variant="listbox" multiple>
                        <flux:option>test</flux:option>
                        <flux:option>dddd</flux:option>
                    </flux:select>
                </div>
                <div class="w-44 p-5">
                    <flux:select variant="listbox" >
                        <flux:option>test</flux:option>
                        <flux:option>dddd</flux:option>
                    </flux:select>
                </div>
            </div>
        </div>
    </div>
```

### How do you expect it to work?

Show in dark mode

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by robs007** [comment]
Yes, it is the same. I missed that note about the forms plugin. I did fix it with adding class=dark:bg-gray-800 dark:border-0.
But I guess I can expect to see other issues if I leave the form plugin enabled.
Thanks

**Comment by jeffchown** [comment]
You're welcome, @robs007. Yes, you will likely run into other issues if you keep tailwindcss/forms.  
Don't forget to close this issue.
